tests/test_mailer.py

```diff
@@ -112,6 +112,12 @@ def test_purge_old_entries(self):
         self.assertNotEqual(MessageLog.objects.filter(result=RESULT_FAILURE).count(), 0)
         self.assertEqual(MessageLog.objects.filter(result=RESULT_SUCCESS).count(), 0)
 
+        with patch.object(mailer.models, 'datetime_now') as datetime_now_patch:
+            datetime_now_patch.return_value = datetime_now() + datetime.timedelta(days=2)
+            call_command('purge_mail_log', '1', '-r', 'failure')
+
+        self.assertEqual(MessageLog.objects.filter(result=RESULT_FAILURE).count(), 0)
```

Comment:
I think a better test for this would ensure that `call_command('purge_mail_log', '1', '-r', 'failure')` is only deleting the failures, and not the successes. Would you be able to modify it that way?

Reply:
I didn't do it in the first place because I didn't want to repeat too many portions of code inside the test function (this is why I personnaly prefer `pytest` over `unittest` for its `parametrized` functionnality!).
I still updated the code for this test to make sure more cases are covered.